content/id/includes/task-tutorial-prereqs.md

```diff
@@ -0,0 +1,5 @@
+Kamu harus memiliki sebuah klaster Kubernetes, dan alat *command-line* harus dikonfigurasi untuk berkomunikasi dengan klaster kamu.
```

Comment:
this tutorial-prereqs file is handled in #20261 
so no need to create new file to reduce conflict

Reply:
deleted

Reply:
If I remove this, the CI will be error